src/CMakeLists.txt

```diff
@@ -34,6 +34,132 @@ if (POLICY CMP0057)
   cmake_policy(SET CMP0057 NEW)
 endif()
 
+# Juce requires either DEBUG or NDEBUG to be defined on MacOS.
```

Comment:
We might need to rebase this file off of develop, since it's changed quite a bit, and I think this PR will revert some of those changes....